roles/matrix-server/templates/synapse/homeserver.yaml.j2

```diff
@@ -476,9 +480,16 @@ trusted_third_party_id_servers:
 
 # Users who register on this homeserver will automatically be joined
 # to these rooms
+{% if matrix_autojoin_rooms is defined %}
```

Comment:
Same here. With a sensible default (empty list), we can skip the `is defined` check and just do something like:

```yaml
{% if matrix_autojoin_rooms|length > 0 %}
auto_join_rooms: {{ matrix_autojoin_rooms|to_yaml }}
```

Well, it's similar..